tests/components/config/test_config_entries.py

```diff
@@ -0,0 +1,295 @@
+"""Test config entries API."""
+
+import asyncio
+from collections import OrderedDict
+from unittest.mock import patch
+
+import pytest
+import voluptuous as vol
+
+from homeassistant import config_entries as core_ce
+from homeassistant.config_entries import ConfigFlowHandler, HANDLERS
+from homeassistant.setup import async_setup_component
+from homeassistant.components.config import config_entries
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def client(hass, test_client):
+    """Fixture that can interact with the config manager API."""
+    hass.loop.run_until_complete(async_setup_component(hass, 'http', {}))
+    hass.loop.run_until_complete(config_entries.async_setup(hass))
+    yield hass.loop.run_until_complete(test_client(hass.http.app))
+
+
+@asyncio.coroutine
+def test_get_entries(hass, client):
+    """Test get entries."""
+    MockConfigEntry(
+       domain='comp',
+       title='Test 1',
+       source='bla'
+    ).add_to_hass(hass)
+    MockConfigEntry(
+       domain='comp2',
+       title='Test 2',
+       source='bla2',
+       state=core_ce.ENTRY_STATE_LOADED,
+    ).add_to_hass(hass)
+    resp = yield from client.get('/api/config/config_entries/entry')
+    assert resp.status == 200
+    data = yield from resp.json()
+    for entry in data:
+        entry.pop('entry_id')
```

Comment:
Can we just add explicit entry_ids to the mock entries and also assert them here?

Reply:
I don't like testing random generated info although I guess that would achieve the same logic. In the end we want to check that all but the `entry_id` are the values that we expect.